point-addition/src/conversion.rs

```diff
@@ -0,0 +1,91 @@
+//! This module implements the conversion from the sum of two elliptic curve points to the sum of
+//! two field elements of the field underlying the elliptic curve
+
+use super::{PointAddition, PointAdditionError};
+use async_trait::async_trait;
+use p256::EncodedPoint;
+use share_conversion_aio::{AdditiveToMultiplicative, MultiplicativeToAdditive};
+use share_conversion_core::fields::{p256::P256, Field};
+
+/// The instance used for converting the curve points
+pub struct Converter<T, U, V>
+where
+    T: AdditiveToMultiplicative<V>,
+    U: MultiplicativeToAdditive<V>,
+    V: Field,
+{
+    /// Performs conversion of addition to multiplication
+    a2m_converter: T,
+    /// Performs conversion of multiplication to addition
+    m2a_converter: U,
+    /// A flag which is used to indicate the party, which has to do an inversion operation
+    negate: bool,
+    /// PhantomData used for the underlying elliptic curve field
+    _field: std::marker::PhantomData<V>,
+}
+
+impl<T, U, V> Converter<T, U, V>
+where
+    T: AdditiveToMultiplicative<V>,
+    U: MultiplicativeToAdditive<V>,
+    V: Field,
+{
+    /// Create a new [Converter] instance
+    pub fn new(a2m_converter: T, m2a_converter: U, negate: bool) -> Self {
+        Self {
+            a2m_converter,
+            m2a_converter,
+            negate,
+            _field: std::marker::PhantomData,
+        }
+    }
+
+    /// Perform the conversion of P = A + B => P_x = a + b
+    ///
+    /// This will convert an elliptic curve point addition to an additive sharing in the underlying
```

Comment:
While I understand what you mean by "convert addition to ...", for outsiders it will be hard to understand because "addition" is a process of adding. 
It may be understood as if "instead of performing addition , we perform additive sharing"

Reply:
thank you